guides/plugins/plugins/storefront/add-custom-javascript.md

```diff
@@ -88,16 +88,35 @@ You can also bind your plugin to a DOM element by providing a css selector:
 
 ```javascript
 // <plugin root>/src/Resources/app/storefront/src/main.js
- // Import all necessary Storefront plugins
- import ExamplePlugin from './example-plugin/example-plugin.plugin';
+// Import all necessary Storefront plugins
+import ExamplePlugin from './example-plugin/example-plugin.plugin';
 
- // Register your plugin via the existing PluginManager
- const PluginManager = window.PluginManager;
- PluginManager.register('ExamplePlugin', ExamplePlugin, '[data-example-plugin]');
+// Register your plugin via the existing PluginManager
+const PluginManager = window.PluginManager;
+PluginManager.register('ExamplePlugin', ExamplePlugin, '[data-example-plugin]');
 ```
 
 In this case the plugin just gets executed if the HTML document contains at least one element with the `data-example-plugin` attribute. You can then use `this.el` inside your plugin to access the DOM element your plugin is bound to.
 
+## Registering an async plugin
+
+You can also register an async JS-plugin. Instead of importing a JS-plugin file at the top of your `main.js` you can provide a dynamic import inside `PluginManager.register()`.
+The import path itself can remain the same as for the synchronous import.
```

Comment:
```suggestion
You can also register an async JS-plugin. Instead of importing a JS-plugin file at the top of your `main.js`, you can provide a dynamic import inside `PluginManager.register()`.
The import path can remain the same as the synchronous import.
```